src/front/glsl/builtins.rs

```diff
@@ -980,6 +980,7 @@ fn inject_common_builtin(
                 let fun = match name {
                     "ceil" => MacroCall::MathFunction(MathFunction::Ceil),
                     "round" => MacroCall::MathFunction(MathFunction::Round),
```

Comment:
nit: was it intentional to not use "|" here?

Reply:
No just me being dumb :)